src/test/java/org/elasticsearch/index/analysis/KuromojiAnalysisTests.java

```diff
@@ -245,4 +245,28 @@ private String readFully(Reader reader) throws IOException {
         return buffer.toString();
     }
 
+    @Test
+    public void testKuromojiUserDict() throws IOException {
+        AnalysisService analysisService = createAnalysisService();
+        TokenizerFactory tokenizerFactory = analysisService.tokenizer("kuromoji_user_dict");
+        String source = "私は制限スピードを超える。";
+        String[] expected = new String[]{"私", "は", "制限スピード", "を", "超える"};
+
+        Tokenizer tokenizer = tokenizerFactory.create();
+        tokenizer.setReader(new StringReader(source));
+        assertSimpleTSOutput(tokenizer, expected);
+    }
+
+    // fix #59
+    @Test
+    public void testKuromojiEmptyUserDict() {
+        AnalysisService analysisService = createAnalysisService();
+        try {
```

Comment:
You probably don't need to try catch here.
If an exception is thrown the test will fail